resolwe_bio/processes/support_processors/seqtk_reverse_complement.py

```diff
@@ -0,0 +1,134 @@
+"""Reverse complement reads with Seqtk."""
+import os
+
+from resolwe.process import Process, DataField, FileField, Cmd, ListField, StringField
+
+
+class ReverseComplementSingle(Process):
+    """Reverse complement FASTQ reads file using Seqtk."""
+
+    slug = 'seqtk-rev-complement-single'
+    process_type = 'data:reads:fastq:single:seqtk:comp'
```

Comment:
I think `data:reads:fastq:single:seqtk` should suffice. Other read processing processes have types:

- `data:reads:fastq:single:bbduk`
- `data:reads:fastq:single:cutadapt`
- ...

The point of process type is to restrict data usage for upstream processes. IMO, one level od depth is enough, but i might be missing something.